components/automate-gateway/handler/iam/v2beta/tokens/tokens.go

```diff
@@ -23,13 +23,20 @@ func NewServer(client authn.TokensMgmtClient) *Server {
 func (s *Server) CreateToken(
 	ctx context.Context, in *pb_req.CreateTokenReq) (*pb_resp.CreateTokenResp, error) {
 
+	// If active value not specified, default to True.
+	var active bool
+	if in.Active == nil {
+		active = true
+	} else {
+		active = in.Active.GetValue()
+	}
```

Comment:
Consider more concise code:
```
active := true
if in.Active != nil {
    active = in.Active.GetValue()
}
```

Reply:
Consider more concise code:
```go
active := in.Active == nil || in.Active.GetValue()
```
just kidding. what we have is fine 👍

Reply:
Actually, I kind of like the single-line. 😀